Branch-SDK/Branch-SDK/BNCServerInterface.m

```diff
@@ -232,6 +232,9 @@ - (NSDictionary *)prepareParamDict:(NSDictionary *)params key:(NSString *)key re
     NSMutableDictionary *fullParamDict = [[NSMutableDictionary alloc] init];
     [fullParamDict addEntriesFromDictionary:params];
     fullParamDict[@"sdk"] = [NSString stringWithFormat:@"ios%@", SDK_VERSION];
+    if ([[[NSBundle mainBundle] executablePath] containsString:@".appex/"]) {
+        fullParamDict[@"ios_extension"] = @(1);
+    }
```

Comment:
It seems like Appled added some build errors to Xcode but you can actually still get the UIApplication stuff and call all of its methods just fine. It'll just behave differently than a normal app. The checks we added elsewhere won't do much it seems so I can go clean them up.